posthog/api/remote_config.py

```diff
@@ -11,28 +13,38 @@ class BaseRemoteConfigAPIView(APIView):
     authentication_classes = []
     permission_classes = []
 
-    def get_object(self, token: str) -> RemoteConfig:
-        try:
-            return RemoteConfig.objects.get(team__api_token=token)
-        except RemoteConfig.DoesNotExist:
-            raise Http404()
+    def check_token(self, token: str):
+        # Check token is a valid token - not longer than 255 characters and only allowing alphanumeric characters and _
+        if len(token) > 200 or not re.match(r"^[a-zA-Z0-9_]+$", token):
```

Comment:
`255` != `200`

Reply:
Ooh yeah comment copy paste error.

For the configs size, we dont store arrayjs - only the config js side of things. So average team config is only a couple of kb max. Could for sure compress it if it seems to be an issue but we already store the team configs in redis via the TeamCache serializer so I don't expect it to be much different.

Reply:
I'll add some metric counters to track this though